extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/ZoneTemperatureControlHandler.java

```diff
@@ -259,9 +260,9 @@ private synchronized BridgeHandler getDssBridgeHandler() {
                 return null;
             }
         }
-        if (temperatureSensorTransmitter == null && zoneID != null) {
-            dssBridgeHandler.registerTemperatureControlStatusListener(this);
-        }
+        // if (temperatureSensorTransmitter == null && zoneID != null) {
```

Comment:
maybe its better to fully remove these lines instead of just commenting them out.